lib/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IDESecureHeapPropagation.cpp

```diff
@@ -35,31 +35,32 @@ IDESecureHeapPropagation::getNormalFlowFunction(n_t Curr, n_t Succ) {
 }
 
 IDESecureHeapPropagation::FlowFunctionPtrType
-IDESecureHeapPropagation::getCallFlowFunction(n_t CallStmt, f_t DestMthd) {
+IDESecureHeapPropagation::getCallFlowFunction(n_t CallSite, f_t DestMthd) {
```

Comment:
Here now all params are upper case, before they started with lower camel. @pdschubert on what style to you want to converge on?

Reply:
Thanks for pointing that out. I'm currently implementing a more rigorous clang-tidy scheme. This will be unified soon.